app/wallets.html

```diff
@@ -45,6 +45,7 @@ <h2>$( wallets.desktop.title )$</h2>
         <a class="d-block" href="https://pokket.cash" target="_blank">Pokket</a>
         <a class="d-block" href="https://melis.io" target="_blank">Melis Wallet</a>
         <a class="d-block" href="https://paytaca.com" target="_blank">Paytaca (Chrome only)</a>
+        <a class="d-block" href="https://flowee.org/products/pay/" target="_blank">Flowee Pay</a>
```

Comment:
Recommend adding (Linux only) after the name

Reply:
https://github.com/bchinfo/bch.info/pull/161/commits/50fea161fb36c21137c41a32060aa0b12e4e491e results in an awkward two line display:
![Screenshot_20220729_034037](https://user-images.githubusercontent.com/34949601/181742899-8b4f2acf-3bf2-4405-bd83-7e9ef566af3e.png)
 
 Can you either move the bracket into \<a> (see Paytaca right above) or into a \<div> (see Paytaca in the mobile section)?